packages/agoric-cli/src/start.js

```diff
@@ -386,10 +386,12 @@ export default async function startMain(progname, rawArgs, powers, opts) {
     const newConfigToml = finishTendermintConfig({
       configToml,
       portNum,
+      enableCors: true,
     });
     const newAppToml = finishCosmosApp({
       appToml,
       portNum,
+      enableCors: true,
```

Comment:
I suspect this CORS stuff is left-over from some tangentially related work. Unless you're clear on why it belongs, perhaps take it out?